packages/shared/src/lib/geolocation.ts

```diff
@@ -545,6 +547,17 @@ const extendedCurrencySymbol = {
     International: '$',
 };
 
+export const isRegion = (val: string): boolean => {
+    return Object.keys(countryGroups).includes(val);
+};
+
+export const getTargetName = (val: string): string => {
+    if (isRegion(val)) {
+        return countryGroups[val].name;
+    }
+    return countryNames[val] || '';
```

Comment:
Would it be clearer to use `undefined` to represent having failed to match a target? It feels more natural to me to use `undefined` for this than an empty string. What do you reckon?

```suggestion
    return countryNames[val];
```

Reply:
I did some investigating and found this:

<img width="710" alt="Screenshot 2023-08-03 at 16 07 32" src="https://github.com/guardian/support-dotcom-components/assets/5357530/106a8ee4-ab4f-4eaa-afee-700dbd2d54f1">

I'm going to try removing this code to see what happens ...